sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-26.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Maintainers Minutes: August 2024'
+subtitle: 'Keeping you informed of the latest maintainers discussions'
+pubDate: 2024-08-14T12:00:00+02:00
+headerImage: '/assets/images/blog/maintainers-minutes-2024-07-26/maintainers_duck.png'
+headerImageAlt: "Cartoon yellow rubber duck with nf-core logo badge on it's body with the nf-core logo."
+embedHeaderImage: true
+authors:
+    - 'jfy133'
+    - 'FriederikeHanssen'
+label:
+    - 'maintainers'
+---
+
+The 'Maintainers Minutes' aims to further give insight into the workings of the [nf-core maintainers team](/governance#maintainers)
+by providing brief summaries of the monthly team meetings.
+
+## Review of action points from last time
+
+We got about half way through our action points from last time.
+
+Slack channels were created for the new working groups: - #wg-meta-pipelines - #wg-modules-update - #wg-test-data-task-force
+
+Please join if you want to be involved in one of those.
+
+## More (or less) testing
+
+Júlia encountered several failing tests on nf-core/crisprseq with conda and singularity. This was caused by tests only running
+on docker. Moving forward we agreed that at least some tests should also be run
+with `singularity` and `conda`.
+
+We discussed several ideas. On the one hand we'd like to make sure everything works at all times, on the other hand running tons of tests
+makes development slow, costs money, and is not great for the planet 🌳. As a compromise, we agreed on the following: - Singularity & conda tests are run on PR to master - Make singularity required - Conda optional opt-out - Add `workflow dispatch` button for easier manual testing
+
+## Nf-test best practices
+
+Nicolas brought up that snapshots should be readable to quickly determine that they do what they should. Since the order of snapshots in the `.snap` file
+does not correspond to the order they are defined in the test file, having one snapshot per file in the same test
+scrambles the entries all over the place.
+
+Nicolas will make updates to the guidelines that all assertions should go into a single snapshot per test.
```

Comment:
```suggestion
Nicolas brought up that nf-test snapshots (i.e., files that record e.g. md5sums or presence of strings) should be readable to quickly determine that they include everything they should.
He had noticed that some people in module tests had started using a snapshot for every single file (rather than all in one), seemingly to be able to 'label' each snapshot.
However, since the order of each 'snapshot' recorded in the `.snap` file does not necessarily correspond to the order they are defined in the test file, having one snapshot _per file_ in the same test scrambles the entries all over the place, e.g. out of order from the output channels as defined in the module.

Nicolas will make updates to the guidelines that all assertions should go into a single snapshot per test for modules.
```